bin/create-moodle-config

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+if [ -f .env ]; 
+then
+    source .env
```

Comment:
This will not work as the variables are not exported (at least inside you sample env file)
Also applies to all following uses of `source .env`